src/components/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { NO_VALUE } from '../../helpers/constants';
+import { findPerson } from '../../helpers/helperFunctions';
+
+type Props = {
+  persons: null | Person[],
+};
+
+export const PersonList: FC<Props> = ({ persons }) => {
+  const { slug } = useParams();
+
+  const tableHeaders = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+
+  if (persons?.length === 0) {
+    return (
+      <p data-cy="noPeopleMessage">
+        There are no people on the server
+      </p>
+    );
+  }
```

Comment:
The component named "list" should render a list, not decide whether it should render a list or not